getnacelle/nacelle-react
# [react-dev-utils]: Add utility package & update dependencies

**Story:** [NC-1677](https://nacelle.atlassian.net/browse/NC-1677) 

### Type of Change


### What is being changed and why?

- Adding a utility package to reduce duplication in dependent packages. This utility package builds relevant types from our GraphQL schemas and exposes them for use in all of our other projects. This reduces the need to continuously create and modify these types locally.



### How to Test

- Running `npm run bootstrap && npm run build` in the root of the monorepo should install the dev utils in the appropriate packages and all builds should succeed
